stdlib/2.7/UserDict.pyi

```diff
@@ -8,4 +9,39 @@ class UserDict(Dict[_KT, _VT], Generic[_KT, _VT]):
 
     def __init__(self, initialdata: Mapping[_KT, _VT] = ...) -> None: ...
 
-    # TODO: DictMixin
+    # TODO: __iter__ is not available for UserDict
+
+class IterableUserDict(UserDict[_KT, _VT], Generic[_KT, _VT]):
+    ...
+
+class DictMixin(Sized, Iterable[_KT], Container[_KT], Generic[_KT, _VT]):
+    @abstractmethod
+    def __getitem__(self, key: _KT) -> _VT: ...
+    @abstractmethod
+    def __setitem__(self, key: _KT, value: _VT) -> None: ...
+    @abstractmethod
+    def __delitem__(self, key: _KT) -> None: ...
+    @abstractmethod
+    def keys(self) -> Iterable[_KT]: ...
```

Comment:
What is the point of these abstract methods? The real DictMixin class is not an ABC and does not define these.